src/Tickets/Commerce/Gateways/Free/Gateway.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * Tickets Commerce: Free Gateway.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Commerce\Gateways\Free
+ */
+
+namespace TEC\Tickets\Commerce\Gateways\Free;
+
+use TEC\Tickets\Commerce\Cart;
+use TEC\Tickets\Commerce\Gateways\Contracts\Abstract_Gateway;
+use TEC\Tickets\Commerce\Module;
+use Tribe__Template as Template;
+
+/**
+ * Class Free Gateway.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Commerce\Gateways\Free
+ */
+class Gateway extends Abstract_Gateway {
+	/**
+	 * The Gateway key.
+	 *
+	 * @since TBD
+	 *
+	 * @var string $key The Gateway key.
+	 */
+	protected static $key = 'free';
+	
+	/**
+	 * @inheritDoc
+	 */
+	public static function get_label() {
+		return '';
+	}
+	
+	/**
+	 * @inheritDoc
+	 */
+	public static function is_enabled(): bool {
```

Comment:
```
	public static function is_enabled(): bool {
```
`is_enabled()` vs `is_active()` is a bit nuanced. This may be a refactor beyond scope, but wanted to mention it at least.

Reply:
Same as this: https://github.com/the-events-calendar/event-tickets/pull/3011#discussion_r1579955231